lib/galaxy/datatypes/binary.py

```diff
@@ -169,6 +169,11 @@ def _get_samtools_version( self ):
                 break
         return version
 
+    @staticmethod
+    def merge(split_files, output_file):
+        params = ["samtools", "merge", "-f", output_file, " ".join(split_files)]
```

Comment:
I was wondering about ``shell=True``, ideally from a security point no of the existing calls in ``binary.py`` would do that either.